lib/msal-core/src/Account.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) Microsoft Corporation
```

Comment:
Use new copyright in all the other files

Reply:
Yes. As I mentioned above, I plan to rebase the final integration branch to dev soon and this should be covered in the same.